# Исследование игровой индустрии в начале XXI века

- Автор: Бутаков Павел Викторович
- Дата: 20.04.2025

### Цели и задачи проекта

Изучить развитие игровой индустрии с 2000 по 2013 год, выявить основные тенденции, включая популярные платформы и жанры, а также влияние оценок критиков и пользователей на продажи игр.

Задачи:

1.  Загрузить и изучить данные об играх.
2.  Провести предобработку данных, включая обработку пропусков, дубликатов и приведение данных к нужным типам.
3.  Отфильтровать данные по периоду с 2000 по 2013 год.
4.  Категоризировать игры по оценкам пользователей и критиков.
5.  Выделить топ-7 платформ по количеству выпущенных игр.</font>

### Описание данных

Данные содержат информацию о продажах игр разных жанров и платформ, а также пользовательские и экспертные оценки игр:
- Name — название игры.
- Platform — название платформы.
- Year of Release — год выпуска игры.
- Genre — жанр игры.
- NA sales — продажи в Северной Америке (в миллионах проданных копий).
- EU sales — продажи в Европе (в миллионах проданных копий).
- JP sales — продажи в Японии (в миллионах проданных копий).
- Other sales — продажи в других странах (в миллионах проданных копий).
- Critic Score — оценка критиков (от 0 до 100).
- User Score — оценка пользователей (от 0 до 10).
- Rating — рейтинг организации ESRB

### Содержимое проекта

1.  Загрузка данных и знакомство с ними.
2.  Проверка ошибок в данных и их предобработка.
3.  Фильтрация данных.
4.  Категоризация данных.
5.  Итоговый вывод.
---

## 1. Загрузка данных и знакомство с ними

- Загрузите необходимые библиотеки Python и данные датасета `/datasets/new_games.csv`.


In [1]:
import pandas as pd

In [2]:
# Загрузка данных
file_path = '/datasets/new_games.csv'
df = pd.read_csv(file_path)

In [3]:
# Первичный обзор
print(df.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 16956 entries, 0 to 16955
Data columns (total 11 columns):
 #   Column           Non-Null Count  Dtype  
---  ------           --------------  -----  
 0   Name             16954 non-null  object 
 1   Platform         16956 non-null  object 
 2   Year of Release  16681 non-null  float64
 3   Genre            16954 non-null  object 
 4   NA sales         16956 non-null  float64
 5   EU sales         16956 non-null  object 
 6   JP sales         16956 non-null  object 
 7   Other sales      16956 non-null  float64
 8   Critic Score     8242 non-null   float64
 9   User Score       10152 non-null  object 
 10  Rating           10085 non-null  object 
dtypes: float64(4), object(7)
memory usage: 1.4+ MB
None


**Познакомимся с данными**

In [4]:
# Получение информации о количестве строк и столбцов
print(df.shape)
# Вывод первых 5 строк датасета
print(df.head())
# Сохраняем количество строк в самом начале
initial_rows = len(df)

(16956, 11)
                       Name Platform  Year of Release         Genre  NA sales  \
0                Wii Sports      Wii           2006.0        Sports     41.36   
1         Super Mario Bros.      NES           1985.0      Platform     29.08   
2            Mario Kart Wii      Wii           2008.0        Racing     15.68   
3         Wii Sports Resort      Wii           2009.0        Sports     15.61   
4  Pokemon Red/Pokemon Blue       GB           1996.0  Role-Playing     11.27   

  EU sales JP sales  Other sales  Critic Score User Score Rating  
0    28.96     3.77         8.45          76.0          8      E  
1     3.58     6.81         0.77           NaN        NaN    NaN  
2    12.76     3.79         3.29          82.0        8.3      E  
3    10.93     3.28         2.95          80.0          8      E  
4     8.89    10.22         1.00           NaN        NaN    NaN  


**Вывод о полученных данных:**

* Было предоставлено 16956 строк и 11 столбцов данных.
* Данные, в целом, соответствуют описанию, однако, наблюдаются пропуски в столбцах `Name`, `Year of Release`, `Genre`, `Critic Score`, `User Score`, `Rating`.
* Типы данных столбцов соответствуют их содержимому, за исключением `Year_of_Release`, который имеет тип `float64`, хотя представляет собой год, и его можно было бы привести к целочисленному типу. Также неверный формат имеют столбцы `EU sales` и `JP sales`, которые имеют тип `object`, и их необходимо перевести к типу `float64`.

**Особенности данных, требующие внимания:**

* Названия столбцов не соответствуют стилю snake\_case (например, `Year of Release`).
* В столбцах `Name`, `Year of Release`, `Genre`, `Critic Score`, `User Score`, `Rating` имеются пропущенные значения.
* Столбец `Year_of_Release` имеет тип данных `float64`, что не оптимально для представления года.

**Дальнейшие шаги предобработки должны включать:**

* Приведение названий столбцов к стилю snake\_case.
* Обработку пропущенных значений.
* Преобразование типа данных столбца `Year_of_Release` в `int64`.

---

## 2.  Проверка ошибок в данных и их предобработка


### 2.1. Названия, или метки, столбцов датафрейма

- Выведите на экран названия всех столбцов датафрейма и проверьте их стиль написания.
- Приведите все столбцы к стилю snake case. Названия должны быть в нижнем регистре, а вместо пробелов — подчёркивания.

In [5]:
# Приведение названий столбцов к snake_case
df.columns = df.columns.str.strip().str.lower().str.replace(' ', '_')
print(df.columns)

Index(['name', 'platform', 'year_of_release', 'genre', 'na_sales', 'eu_sales',
       'jp_sales', 'other_sales', 'critic_score', 'user_score', 'rating'],
      dtype='object')


### 2.2. Типы данных

- Если встречаются некорректные типы данных, предположите их причины.
- При необходимости проведите преобразование типов данных. Помните, что столбцы с числовыми данными и пропусками нельзя преобразовать к типу `int64`. Сначала вам понадобится обработать пропуски, а затем преобразовать типы данных.

In [6]:
# Вывод информации о типах данных
print("Типы данных до преобразования:")
print(df.dtypes)

Типы данных до преобразования:
name                object
platform            object
year_of_release    float64
genre               object
na_sales           float64
eu_sales            object
jp_sales            object
other_sales        float64
critic_score       float64
user_score          object
rating              object
dtype: object


In [7]:
# Вывод уникальных значений перед преобразованием
columns_to_check = ['eu_sales', 'jp_sales', 'user_score']
for col in columns_to_check:
    print(f"\nУникальные значения в столбце '{col}':")
    print(df[col].unique())


Уникальные значения в столбце 'eu_sales':
['28.96' '3.58' '12.76' '10.93' '8.89' '2.26' '9.14' '9.18' '6.94' '0.63'
 '10.95' '7.47' '6.18' '8.03' '4.89' '8.49' '9.09' '0.4' '3.75' '9.2'
 '4.46' '2.71' '3.44' '5.14' '5.49' '3.9' '5.35' '3.17' '5.09' '4.24'
 '5.04' '5.86' '3.68' '4.19' '5.73' '3.59' '4.51' '2.55' '4.02' '4.37'
 '6.31' '3.45' '2.81' '2.85' '3.49' '0.01' '3.35' '2.04' '3.07' '3.87'
 '3.0' '4.82' '3.64' '2.15' '3.69' '2.65' '2.56' '3.11' '3.14' '1.94'
 '1.95' '2.47' '2.28' '3.42' '3.63' '2.36' '1.71' '1.85' '2.79' '1.24'
 '6.12' '1.53' '3.47' '2.24' '5.01' '2.01' '1.72' '2.07' '6.42' '3.86'
 '0.45' '3.48' '1.89' '5.75' '2.17' '1.37' '2.35' '1.18' '2.11' '1.88'
 '2.83' '2.99' '2.89' '3.27' '2.22' '2.14' '1.45' '1.75' '1.04' '1.77'
 '3.02' '2.75' '2.16' '1.9' '2.59' '2.2' '4.3' '0.93' '2.53' '2.52' '1.79'
 '1.3' '2.6' '1.58' '1.2' '1.56' '1.34' '1.26' '0.83' '6.21' '2.8' '1.59'
 '1.73' '4.33' '1.83' '0.0' '2.18' '1.98' '1.47' '0.67' '1.55' '1.91'
 '0.69' '0.6' '1.93' '1.64' 

In [8]:
# Преобразование столбцов
df['year_of_release'] = pd.to_numeric(df['year_of_release'], errors='coerce').fillna(0).astype('int64')
df['eu_sales'] = pd.to_numeric(df['eu_sales'], errors='coerce')
df['jp_sales'] = pd.to_numeric(df['jp_sales'], errors='coerce')
df['user_score'] = pd.to_numeric(df['user_score'], errors='coerce')

- В числовых столбцах могут встретиться строковые значения, например `unknown` или другие. Приводите такие столбцы к числовому типу данных, заменив строковые значения на пропуски.

In [9]:
# Вывод информации о типах данных после преобразования
print("\nТипы данных после преобразования:")
print(df.dtypes)


Типы данных после преобразования:
name                object
platform            object
year_of_release      int64
genre               object
na_sales           float64
eu_sales           float64
jp_sales           float64
other_sales        float64
critic_score       float64
user_score         float64
rating              object
dtype: object


### 2.3. Наличие пропусков в данных

- Посчитайте количество пропусков в каждом столбце в абсолютных и относительных значениях.


In [10]:
# Подсчет пропусков
total_missings = df.isnull().sum().sort_values(ascending=False)
percentage_missings = (df.isnull().sum() / df.isnull().count()).sort_values(ascending=False)

missing_data = pd.concat([total_missings, percentage_missings], axis=1, keys=['Total', 'Percent'])

print("Пропуски до обработки:")
print(missing_data)

# Функция для заполнения пропусков в продажах
def fill_sales_na(df, col):
    df[col] = df[col].fillna(df.groupby(['platform', 'year_of_release'])[col].transform('mean'))
    return df

# Обработка пропусков в eu_sales и jp_sales
df = fill_sales_na(df, 'eu_sales')
df = fill_sales_na(df, 'jp_sales')

# Заполнение оставшихся пропусков в продажах (если есть) медианой по всему столбцу
df['eu_sales'].fillna(df['eu_sales'].median(), inplace=True)
df['jp_sales'].fillna(df['jp_sales'].median(), inplace=True)

# Пересчет пропусков после обработки
total_missings_after = df.isnull().sum().sort_values(ascending=False)
percentage_missings_after = (df.isnull().sum() / df.isnull().count()).sort_values(ascending=False)

missing_data_after = pd.concat([total_missings_after, percentage_missings_after], axis=1, keys=['Total', 'Percent'])

print("\nПропуски после обработки:")
print(missing_data_after)

Пропуски до обработки:
                 Total   Percent
user_score        9268  0.546591
critic_score      8714  0.513918
rating            6871  0.405225
eu_sales             6  0.000354
jp_sales             4  0.000236
name                 2  0.000118
genre                2  0.000118
platform             0  0.000000
year_of_release      0  0.000000
na_sales             0  0.000000
other_sales          0  0.000000

Пропуски после обработки:
                 Total   Percent
user_score        9268  0.546591
critic_score      8714  0.513918
rating            6871  0.405225
name                 2  0.000118
genre                2  0.000118
platform             0  0.000000
year_of_release      0  0.000000
na_sales             0  0.000000
eu_sales             0  0.000000
jp_sales             0  0.000000
other_sales          0  0.000000


- Изучите данные с пропущенными значениями. Напишите промежуточный вывод: для каких столбцов характерны пропуски и сколько их. Предположите, почему пропуски могли возникнуть. Укажите, какие действия с этими данными можно сделать и почему.


**Предположения о причинах пропусков:**

* `rating`:  Отсутствие рейтинга может быть связано с тем, что игра не была оценена ESRB, либо данные не были внесены.
* `critic_score`, `user_score`: Пропуски в оценках могут быть связаны с тем, что на игру не было достаточного количества обзоров или оценок пользователей, либо данные не были собраны.
* `name`, `genre`:  Пропуски в названии и жанре могут быть связаны с ошибками при сборе данных.

**Действия с пропусками:**

* `rating`: Заменить на "Unknown", так как это категориальный столбец и пропуски составляют значительную часть данных.
* `critic_score`, `user_score`: Оставить как есть, так как заполнение средними значениями может исказить анализ.
* `name`, `genre`: Удалить строки с пропусками, так как их количество незначительно.

- Обработайте пропущенные значения. Для каждого случая вы можете выбрать оптимальный, на ваш взгляд, вариант: заменить на определённое значение, оставить как есть или удалить.
- Если вы решите заменить пропуски на значение-индикатор, то убедитесь, что предложенное значение не может быть использовано в данных.
- Если вы нашли пропуски в данных с количеством проданных копий игры в том или ином регионе, их можно заменить на среднее значение в зависимости от названия платформы и года выхода игры.

In [11]:
# Удаление строк с пропусками в `name` и `genre`
df.dropna(subset=['name', 'genre'], inplace=True)

In [12]:
# Заполнение пропусков в `rating` значением 'Unknown'
df['rating'].fillna('Unknown', inplace=True)

In [13]:
# Проверка обработки пропусков
print(df.isnull().sum())

name                  0
platform              0
year_of_release       0
genre                 0
na_sales              0
eu_sales              0
jp_sales              0
other_sales           0
critic_score       8712
user_score         9266
rating                0
dtype: int64


### 2.4. Явные и неявные дубликаты в данных

- Изучите уникальные значения в категориальных данных, например с названиями жанра игры, платформы, рейтинга и года выпуска. Проверьте, встречаются ли среди данных неявные дубликаты, связанные с опечатками или разным способом написания.
- При необходимости проведите нормализацию данных с текстовыми значениями. Названия или жанры игр можно привести к нижнему регистру, а названия рейтинга — к верхнему.

In [14]:
# Проверка уникальных значений в категориальных столбцах
categorical_columns = ['platform', 'genre', 'rating']
for col in categorical_columns:
    unique_values = df[col].unique()
    print(f"Уникальные значения в столбце '{col}':")
    print(unique_values)

Уникальные значения в столбце 'platform':
['Wii' 'NES' 'GB' 'DS' 'X360' 'PS3' 'PS2' 'SNES' 'GBA' 'PS4' '3DS' 'N64'
 'PS' 'XB' 'PC' '2600' 'PSP' 'XOne' 'WiiU' 'GC' 'GEN' 'DC' 'PSV' 'SAT'
 'SCD' 'WS' 'NG' 'TG16' '3DO' 'GG' 'PCFX']
Уникальные значения в столбце 'genre':
['Sports' 'Platform' 'Racing' 'Role-Playing' 'Puzzle' 'Misc' 'Shooter'
 'Simulation' 'Action' 'Fighting' 'Adventure' 'Strategy' 'MISC'
 'ROLE-PLAYING' 'RACING' 'ACTION' 'SHOOTER' 'FIGHTING' 'SPORTS' 'PLATFORM'
 'ADVENTURE' 'SIMULATION' 'PUZZLE' 'STRATEGY']
Уникальные значения в столбце 'rating':
['E' 'Unknown' 'M' 'T' 'E10+' 'K-A' 'AO' 'EC' 'RP']


In [15]:
# Нормализация столбца 'genre'
df['genre'] = df['genre'].str.lower()

# Нормализация столбца 'rating'
df['rating'] = df['rating'].str.upper()

# Проверка изменений
for col in categorical_columns:
    unique_values = df[col].unique()
    print(f"Уникальные значения в столбце '{col}' после нормализации:")
    print(unique_values)

Уникальные значения в столбце 'platform' после нормализации:
['Wii' 'NES' 'GB' 'DS' 'X360' 'PS3' 'PS2' 'SNES' 'GBA' 'PS4' '3DS' 'N64'
 'PS' 'XB' 'PC' '2600' 'PSP' 'XOne' 'WiiU' 'GC' 'GEN' 'DC' 'PSV' 'SAT'
 'SCD' 'WS' 'NG' 'TG16' '3DO' 'GG' 'PCFX']
Уникальные значения в столбце 'genre' после нормализации:
['sports' 'platform' 'racing' 'role-playing' 'puzzle' 'misc' 'shooter'
 'simulation' 'action' 'fighting' 'adventure' 'strategy']
Уникальные значения в столбце 'rating' после нормализации:
['E' 'UNKNOWN' 'M' 'T' 'E10+' 'K-A' 'AO' 'EC' 'RP']


- После того как нормализуете данные и устраните неявные дубликаты, проверьте наличие явных дубликатов в данных.

In [16]:
# Проверка на явные дубликаты
print("Количество явных дубликатов:", df.duplicated().sum())

Количество явных дубликатов: 241


In [17]:
# Удаление явных дубликатов
df.drop_duplicates(inplace=True)

In [18]:
# Проверка удаления дубликатов
print("Количество явных дубликатов после удаления:", df.duplicated().sum())

Количество явных дубликатов после удаления: 0


- Напишите промежуточный вывод: укажите количество найденных дубликатов и действия по их обработке.

* Было обнаружено 241 явных дубликатов.
* Все явные дубликаты были удалены из данных.
* Неявные дубликаты в столбцах `genre` и `rating` были обработаны путем приведения всех значений к нижнему и верхнему регистру соответственно.

- В процессе подготовки данных вы могли что-либо удалять, например строки с пропусками или ошибками, дубликаты и прочее. В этом случае посчитайте количество удалённых строк в абсолютном и относительном значениях.

In [19]:
# Расчет количества удаленных строк
final_rows = len(df)
deleted_rows = initial_rows - final_rows
deleted_percentage = (deleted_rows / initial_rows) * 100 if initial_rows else 0

print(f"Удалено строк: {deleted_rows} ({deleted_percentage:.2f}%)")

Удалено строк: 243 (1.43%)


- После проведения предобработки данных напишите общий промежуточный вывод.

* Данные были загружены и изучены.
* Названия столбцов приведены к snake\_case.
* Тип данных столбца `year_of_release` преобразован в `int64`.
* Пропущенные значения обработаны: строки с пропусками в `name` и `genre` удалены, пропуски в `rating` заполнены значением 'UNKNOWN', пропуски в `critic_score` и `user_score` оставлены без изменений.
* Явные и неявные дубликаты удалены.
* Общее количество удаленных строк: 243 (1.43%).

---

## 3. Фильтрация данных

Коллеги хотят изучить историю продаж игр в начале XXI века, и их интересует период с 2000 по 2013 год включительно. Отберите данные по этому показателю. Сохраните новый срез данных в отдельном датафрейме, например `df_actual`.

In [20]:
# Фильтрация данных по year_of_release
df_actual = df[(df['year_of_release'] >= 2000) & (df['year_of_release'] <= 2013)].copy()

In [21]:
# Проверка фильтрации
print(df_actual['year_of_release'].min())
print(df_actual['year_of_release'].max())

2000
2013


---

## 4. Категоризация данных
    
Проведите категоризацию данных:
- Разделите все игры по оценкам пользователей и выделите такие категории: высокая оценка (от 8 до 10 включительно), средняя оценка (от 3 до 8, не включая правую границу интервала) и низкая оценка (от 0 до 3, не включая правую границу интервала).

In [22]:
def categorize_user_score(score):
    if pd.isna(score):
        return 'no data'  # Категория для пропущенных значений
    elif 8 <= score <= 10:
        return 'high'
    elif 3 <= score < 8:
        return 'average'
    elif 0 <= score < 3:
        return 'low'
    else:
        return 'unknown'  # Категория для неожиданных значений

In [23]:
df_actual['user_score_category'] = df_actual['user_score'].apply(categorize_user_score)

In [24]:
# Проверка категоризации
print(df_actual['user_score_category'].value_counts())

no data    6298
average    4081
high       2286
low         116
Name: user_score_category, dtype: int64


- Разделите все игры по оценкам критиков и выделите такие категории: высокая оценка (от 80 до 100 включительно), средняя оценка (от 30 до 80, не включая правую границу интервала) и низкая оценка (от 0 до 30, не включая правую границу интервала).

In [25]:
# Категоризация по critic_score
def categorize_critic_score(score):
    if pd.isna(score):
        return 'no data'  # Категория для пропущенных значений
    elif 80 <= score <= 100:
        return 'high'
    elif 30 <= score < 80:
        return 'average'
    elif 0 <= score < 30:
        return 'low'
    else:
        return 'unknown'  # Категория для неожиданных значений

In [26]:
df_actual['critic_score_category'] = df_actual['critic_score'].apply(categorize_critic_score)

In [27]:
# Проверка категоризации
print(df_actual['critic_score_category'].value_counts())

no data    5612
average    5422
high       1692
low          55
Name: critic_score_category, dtype: int64


- После категоризации данных проверьте результат: сгруппируйте данные по выделенным категориям и посчитайте количество игр в каждой категории.

In [28]:
# Группировка и подсчет количества игр по категориям
user_score_counts = df_actual.groupby('user_score_category').size()
critic_score_counts = df_actual.groupby('critic_score_category').size()

In [29]:
print("Распределение игр по категориям оценок пользователей:\n", user_score_counts)
print("\nРаспределение игр по категориям оценок критиков:\n", critic_score_counts)

Распределение игр по категориям оценок пользователей:
 user_score_category
average    4081
high       2286
low         116
no data    6298
dtype: int64

Распределение игр по категориям оценок критиков:
 critic_score_category
average    5422
high       1692
low          55
no data    5612
dtype: int64


- Выделите топ-7 платформ по количеству игр, выпущенных за весь актуальный период.

In [30]:
# Подсчет количества игр по платформам
platform_counts = df_actual['platform'].value_counts().head(7)

In [31]:
print("\nТоп-7 платформ по количеству выпущенных игр:\n", platform_counts)


Топ-7 платформ по количеству выпущенных игр:
 PS2     2127
DS      2120
Wii     1275
PSP     1180
X360    1121
PS3     1087
GBA      811
Name: platform, dtype: int64


---

## 5. Итоговый вывод

В конце напишите основной вывод и отразите, какую работу проделали. Не забудьте указать описание среза данных и новых полей, которые добавили в исходный датасет.

В ходе данного исследования была проанализирована информация о продажах видеоигр с целью выявления ключевых тенденций в игровой индустрии в начале XXI века (2000-2013 годы).

**Проделанная работа:**

1.  Загрузка и изучение данных:
    * Данные были загружены в DataFrame.
    * Проведена первичная оценка структуры данных, типов столбцов и наличия пропусков.
2.  Предобработка данных:
    * Названия столбцов приведены к snake\_case.
    * Тип данных столбца `year_of_release` преобразован в `int64`.
    * Обработаны пропущенные значения: удалены строки с пропусками в столбцах `name` и `genre`, пропуски в столбце `rating` заполнены значением 'UNKNOWN', пропуски в столбцах `critic_score` и `user_score` оставлены без изменений.
    * Устранены явные и неявные дубликаты.
3.  Фильтрация данных:
    * Данные отфильтрованы по периоду с 2000 по 2013 год включительно.
4.  Категоризация данных:
    * Игры категоризированы по оценкам пользователей (`user_score_category`) на 'high', 'average', 'low' и 'no data'.
    * Игры категоризированы по оценкам критиков (`critic_score_category`) на 'high', 'average', 'low' и 'no data'.
    * Определены топ-7 платформ по количеству выпущенных игр.

**Описание среза данных и новых полей:**

* Срез данных (`df_actual`) содержит информацию об играх, выпущенных в период с 2000 по 2013 год.
* Добавлены два новых столбца:
    * `user_score_category`: категория оценки пользователей ('high', 'average', 'low', 'no data').
    * `critic_score_category`: категория оценки критиков ('high', 'average', 'low', 'no data').

**Основные выводы:**

1.  **Объем данных и пропуски:**

    * Было обработано 16956 игр, выпущенных с 2000 по 2013 год.
    * Значительное количество пропусков наблюдается в данных об оценках (`critic_score`, `user_score`) и рейтингах (`rating`), что может ограничить полноту анализа, связанного с этими параметрами.

2.  **Топ-7 платформ:**

    * Наибольшее количество игр за рассматриваемый период было выпущено на следующих платформах (в порядке убывания): PS2 - 2127, DS - 2120, Wii - 1275, PSP - 1180, X360 - 1121, PS3 - 1087, GBA - 811. Это свидетельствует о доминировании этих платформ на рынке в начале 2000-х.

3.  **Распределение оценок:**

    * Большинство игр имеют средние оценки как от пользователей, так и от критиков. Это может говорить о том, что основная масса выпускаемых игр была "середнячками", без ярко выраженных провалов или хитов.
    * Необходимо учитывать, что значительная доля данных об оценках отсутствует, что может смещать распределение.

4.  **Потенциальные направления дальнейшего анализа:**

    * **Влияние платформы на продажи:** Интересно было бы исследовать, как продажи игр (в разных регионах) коррелируют с платформой выпуска. Можно посмотреть, на каких платформах были самые высокие средние продажи.
    * **Жанровые предпочтения:** Анализ распределения игр и их продаж по жанрам позволит выявить наиболее популярные жанры в рассматриваемый период.
    * **Динамика продаж во времени:** Можно посмотреть, как менялись объемы продаж и предпочтения игроков с течением времени (2000-2013).
    * **Региональные особенности:** Сравнение продаж в разных регионах (NA, EU, JP, Other) позволит выявить региональные различия в предпочтениях игроков.